docs/samples/autoscaling/README.md

```diff
@@ -0,0 +1,306 @@
+```
+$ kubectl get pods
+NAME                                                       READY   STATUS            RESTARTS   AGE
+flowers-sample-default-7kqt6-deployment-75d577dcdb-sr5wd         3/3     Running       0          42s
+flowers-sample-default-7kqt6-deployment-75d577dcdb-swnk5         3/3     Running       0          62s
+flowers-sample-default-7kqt6-deployment-75d577dcdb-t2njf         3/3     Running       0          62s
+flowers-sample-default-7kqt6-deployment-75d577dcdb-vdlp9         3/3     Running       0          64s
+flowers-sample-default-7kqt6-deployment-75d577dcdb-vm58d         3/3     Running       0          42s
+```
+
+### Check Dashboard
+View the Knative Serving Scaling dashboards (if configured).
+
+```bash
+kubectl port-forward --namespace knative-monitoring $(kubectl get pods --namespace knative-monitoring --selector=app=grafana  --output=jsonpath="{.items..metadata.name}") 3000
+```
+
+![scaling dashboard](scaling_debug.png)
+
+
+## Load your KFService with target QPS
+
+### Create the KFService
+Apply the tensorflow example CR
+```
+kubectl apply -f autoscale.yaml
+```
+
+Expected Output
+```
+$ kfservice.serving.kubeflow.org/flowers-sample configured
+```
+
+### Load KFService with target QPS
+Send 30 seconds of traffic maintaining 50 qps.
+```bash
+MODEL_NAME=flowers-sample
+INPUT_PATH=../tensorflow/input.json
+CLUSTER_IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+HOST=$(kubectl get kfservice $MODEL_NAME -o jsonpath='{.status.url}')
+hey -z 30s -q 50 -host ${HOST} -D $INPUT_PATH http://$CLUSTER_IP/v1/models/$MODEL_NAME:predict
+```
+
+```shell
+Summary:
+  Total:	30.0264 secs
+  Slowest:	10.8113 secs
+  Fastest:	0.0145 secs
+  Average:	0.0731 secs
+  Requests/sec:	683.5644
+  
+  Total data:	5069675 bytes
+  Size/request:	247 bytes
+
+Response time histogram:
+  0.014 [1]	|
+  1.094 [20474]	|■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
+  2.174 [0]	|
+  3.254 [0]	|
+  4.333 [0]	|
+  5.413 [0]	|
+  6.493 [0]	|
+  7.572 [0]	|
+  8.652 [0]	|
+  9.732 [0]	|
+  10.811 [50]	|
+
+
+Latency distribution:
+  10% in 0.0284 secs
+  25% in 0.0334 secs
+  50% in 0.0408 secs
+  75% in 0.0527 secs
+  90% in 0.0765 secs
+  95% in 0.0949 secs
+  99% in 0.1334 secs
+
+Details (average, fastest, slowest):
+  DNS+dialup:	0.0001 secs, 0.0145 secs, 10.8113 secs
+  DNS-lookup:	0.0000 secs, 0.0000 secs, 0.0196 secs
+  req write:	0.0000 secs, 0.0000 secs, 0.0031 secs
+  resp wait:	0.0728 secs, 0.0144 secs, 10.7688 secs
+  resp read:	0.0000 secs, 0.0000 secs, 0.0031 secs
+
+Status code distribution:
+  [200]	20525 responses
+```
+
+Check the number of pods now, we are loading the service with 50 requests per second, and from the dashboard you can see
+that it hits the average concurrency 10 and autoscaler tries scaling up to 10 pods.
+
```

Comment:
Check the number of running pods now. We are loading the service with 50 requests per second, and from the dashboard you can see that it hits the average concurrency of 10 and autoscaler tries scaling up to 10 pods.